test/govspeak_contacts_test.rb

```diff
@@ -74,17 +96,11 @@ def compress_html(html)
   end
 
   test "contact with no postal address omits the address info" do
-    contact = build_contact(
-      recipient: nil,
-      street_address: nil,
-      locality: nil,
-      region: nil,
-      postal_code: nil,
-    )
+    contact = build_contact(post_addresses: [])
```

Comment:
Not really part of this PR, but I notice there's (effectively) a duplicate test about not finding a contact, and no test for the expected HTML when there's no email/phone/URL.

Reply:
https://github.com/alphagov/govspeak/blob/ec96ea5cfb334c814232f17f1bdc71b446f8254b/test/govspeak_contacts_test.rb#L86-L96